policy/modules/system/systemd.te

```diff
@@ -1178,6 +1185,14 @@ allow systemd_tmpfiles_t systemd_journal_t:file relabel_file_perms;
 allow systemd_tmpfiles_t systemd_tmpfiles_conf_t:dir list_dir_perms;
 allow systemd_tmpfiles_t systemd_tmpfiles_conf_type:file read_file_perms;
 
+allow systemd_tmpfiles_t systemd_pstore_var_lib_t:dir { relabelfrom relabelto manage_dir_perms };
```

Comment:
CI had an error, the `relabelfrom relabelto` should be replaced with `relabel_dir_perms`.